### ISSUE TITLE
iPhone X support

### ISSUE THREAD

**Issue by getaaron** [thiagoperes/IDMPhotoBrowser#271]
When presenting the `IDMPhotoBrowser`, the "Done" button is obscured by the sensor housing notch:

![image](https://user-images.githubusercontent.com/789577/31060925-dc68761a-a6e0-11e7-823f-c6cb146c8aed.png)

The top half of the "Done" button doesn't respond to touch events.

**Comment by PrakashMaharjan** [comment]
I just checked if the device is iphone X or not set the top inset of done button as shown below:

browser.doneButtonTopInset = 50.0f;

This will shift the done button little below.

**Comment by getaaron** [comment]
@PrakashMaharjan Right, but ideally the `IDMPhotoBrowser` would respect the safe insets property

**Comment by getaaron** [comment]
I went with this for now…

```objective-c
if (@available(iOS 11.0, *)) {
    browser.doneButtonTopInset = MAX(browser.doneButtonTopInset, viewController.view.safeAreaInsets.top);
}
```

A more robust improvement that used the auto-layout constraints would be preferred.

**Comment by chris-hut** [comment]
Isn't this closed by https://github.com/thiagoperes/IDMPhotoBrowser/pull/282

**Comment by SDGMazeGeek** [comment]
is this solved yet?
